Contents/Code/plex_api_helper.py

```diff
@@ -193,7 +193,7 @@ def update_plex_item(rating_key):
                                     except Exception as e:
                                         Log.Error('{}: Error updating summary: {}'.format(item.ratingKey, e))
 
-                if item.isLocked(field='theme'):
+                if item.isLocked(field='theme') and not os.environ.get('THEMERR_CI_TESTING'):
```

Comment:
```suggestion
                if item.isLocked(field='theme'):
```

environment variable was not detected by plugin